util/network-devp2p/src/host.rs

```diff
@@ -1099,7 +1099,7 @@ fn save_key(path: &Path, key: &Secret) {
 	if let Err(e) = restrict_permissions_owner(path, true, false) {
 		warn!(target: "network", "Failed to modify permissions of the file ({})", e);
 	}
-	if let Err(e) = file.write(&key.hex().into_bytes()) {
+	if let Err(e) = file.write(&key.hex().into_bytes()[2..]) {
```

Comment:
Was this needed or was changed just for "style"? I.e. will `Secret::from_str` work with and without the `0x` prefix?

Reply:
That was required, test caught it.